distribution/lib/Standard/Base/0.0.0-dev/src/Data/Json/Extensions.enso

```diff
@@ -135,3 +135,10 @@ Locale.to_js_object self =
     b.append ["country", self.country]
     b.append ["variant", self.variant]
     JS_Object.from_pairs b.to_vector
+
+## Converts the given value to a JSON serializable object.
+   For Map, this serialized as a Vector of Key-Value pairs.
+Map.to_js_object : JS_Object
+Map.to_js_object self =
+    map_vector = self.to_vector
+    map_vector.map p-> [p.first.to_js_object, p.second.to_js_object]
```

Comment:
btw. why do we serialize `Map` as a vector of kv-pairs instead of a JS object?

Reply:
JS object can only have string keys.

Reply:
Yeah, I remembered that and added a comment above ^